### DISCUSSION TITLE
Wave DB binary for ARM64

### DISCUSSION THREAD

**Discussion by sulhicader** [h2oai/wave#2374] [General]
Hi Team, where I can find the WaveDB binary for ARM64(M1). When I tried with available binary i am getting following error.

/bin/sh: /Users/sulhicader/Desktop/h2oai/dev/wave-h2o-drive/wavedb: cannot execute binary file

**Comment by mturoci** [comment]
There is no wavedb arm build atm, feel free to file an issue though.
